app/locale/en.coffee

```diff
@@ -3163,3 +3165,9 @@ module.exports = nativeDescription: "English", englishDescription: "English", tr
     pitch_body: 'For the best CodeCombat experience we recommend using the latest version of Chrome. Download the latest version of chrome by clicking the button below!'
     download: 'Download Chrome'
     ignore: 'Ignore'
+
+  admin:
+    license_type_full: 'Full Courses'
+    license_type_preset1: 'CS1+CS2+GD1+WD1'
```

Comment:
```suggestion
    license_type_preset1: 'CS1+CS2+GD1+WD1 (Starter Licenses)'
```

Reply:
@nwinter i think.. the `license_type_preset1` is also a customized license although it has the same courses as starter license. will this commit makes things confusing?